apps/cms.c

```diff
@@ -22,6 +22,8 @@
 # include <openssl/x509_vfy.h>
 # include <openssl/x509v3.h>
 # include <openssl/cms.h>
+# include <openssl/ts.h>
+# include "crypto/ts/ts_lcl.h"
```

Comment:
This includes fixed m previous comment, but not in the way !
Structures are now **Opaque** : it means that client code  (like this app) just handle pointers on a type, without knowing structure internals: whole size, which fields, etc...
Solution: see my bottom comment